src/vision/src/HSVFilterNode.cpp

```diff
@@ -6,23 +6,81 @@
 
 #include <HSVFilterNode.h>
 
+/*
+ * This overrides the default method. Upon killing the program
+ * Run custom code to update the dynamic simulation parameter values into the
+ * wroking yaml file.
+ */
+void mySigintHandler(int sig) {
+    std::cout << "\nUpdating the yaml files... preparing for shutdown.\n"
```

Comment:
nit, but don't need the last `\n` here since we already have the `std::endl`

Reply:
Done